lib/vrl/stdlib/src/sha2.rs

```diff
@@ -8,8 +8,8 @@ fn sha2(value: Value, variant: &Bytes) -> Resolved {
         b"SHA-256" => encode::<Sha256>(&value),
         b"SHA-384" => encode::<Sha384>(&value),
         b"SHA-512" => encode::<Sha512>(&value),
-        b"SHA-512/224" => encode::<Sha512Trunc224>(&value),
-        b"SHA-512/256" => encode::<Sha512Trunc256>(&value),
+        b"SHA-512/224" => encode::<Sha512_224>(&value),
+        b"SHA-512/256" => encode::<Sha512_256>(&value),
```

Comment:
Crate upgrade brought in a type name change?

Reply:
Yeah, unfortunately.